iOS/CodiumUnWrapper/CodiumUnWrapper.swift

```diff
@@ -0,0 +1,155 @@
+// Codium UnWrapper
+
+import Foundation
+import UIKit
+
+extension Optional {
+
+    func unWrapOr<T>(_ value : T) -> T {
```

Comment:
`unwrap` is one word

Reply:
I changed it to `unwrap` I will push again